.travis.yml

```diff
@@ -31,8 +31,9 @@ before_install:
   - sudo apt-get -q update
 
 install:
-  - sudo apt-get install -y python
+  - sudo apt-get install -y python python3 python3-pip
```

Comment:
Done. `python2` is not a package name, but I changed `python` to `python2` in other places.